buildsystem/dependencies.gradle

```diff
@@ -0,0 +1,59 @@
+allprojects {
+    repositories {
+        jcenter()
+    }
+}
+
+ext {
```

Comment:
I don't see the Kotlin version definition

Reply:
done!